HPXMLtoOpenStudio/measure.rb

```diff
@@ -4169,15 +4191,102 @@ def self.set_surface_exterior(model, spaces, surface, exterior_adjacent_to)
     elsif [HPXML::LocationBasementConditioned].include? exterior_adjacent_to
       surface.createAdjacentSurface(create_or_get_space(model, spaces, HPXML::LocationLivingSpace))
       @cond_bsmnt_surfaces << surface
+    elsif [HPXML::LocationOtherHeatedSpace, HPXML::LocationOtherMultifamilyBufferSpace, HPXML::LocationOtherNonFreezingSpace].include? exterior_adjacent_to
+      add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+      surface.setSunExposure('NoSun')
+      surface.setWindExposure('NoWind')
```

Comment:
Minor suggestion, but it seems like these should be moved inside of add_otherside_coefficients, since you'd never want to call that method without also setting these.